t/15_qesap_azure.t

```diff
@@ -146,4 +146,80 @@ subtest '[qesap_az_vnet_peering_delete] delete failure' => sub {
     ok((any { /jsc#7487/ } @soft_failure), 'soft failure');
 };
 
+subtest '[qesap_az_get_tenant_id]' => sub {
```

Comment:
can be potentially split in two tests

Reply:
Done. 
I have created a separate uuid regex check function and both qesap_az_enable_system_assigned_identity and qesap_az_get_tenant_id are now using it :

https://github.com/os-autoinst/os-autoinst-distri-opensuse/compare/master...lpalovsky:os-autoinst-distri-opensuse:Azure-MSI-based-fencing#diff-4b8cb2b3db3a3b1a28d8774fe6e830e76c20a4c68e3e12bb862398ad9f4cc5b2R190

https://github.com/os-autoinst/os-autoinst-distri-opensuse/compare/master...lpalovsky:os-autoinst-distri-opensuse:Azure-MSI-based-fencing#diff-4b8cb2b3db3a3b1a28d8774fe6e830e76c20a4c68e3e12bb862398ad9f4cc5b2R206

https://github.com/os-autoinst/os-autoinst-distri-opensuse/compare/master...lpalovsky:os-autoinst-distri-opensuse:Azure-MSI-based-fencing#diff-4b8cb2b3db3a3b1a28d8774fe6e830e76c20a4c68e3e12bb862398ad9f4cc5b2R219